packages/editor/src/components/visual-editor/style.scss

```diff
@@ -1,8 +1,19 @@
 .editor-visual-editor {
 	position: relative;
 	display: flex;
+
+	// This duplicates the iframe background but it's necessary in some situations
+	// when the iframe doesn't cover the whole canvas
+	// like the "focused entities".
 	background-color: $gray-300;
```

Comment:
If we have this rule, why do we still need the background in `packages/block-editor/src/components/block-canvas/style.scss`?

Reply:
For third-party block editor usage.